lib/chef-dk/skeletons/code_generator/files/default/delivery-project.toml

```diff
@@ -29,8 +26,9 @@ cleanup = "chef exec kitchen destroy"
 
 # Remote project.toml file
 #
-# Specify a remote URI location for the `project.toml` file.
-# This is useful for teams that wish to centrally manage the behavior
-# of the `delivery local` command across many different projects.
+# Instead of the local phases above you may specify a remote URI location for
```

Comment:
So, there is a comma here that I want to insert, but I know this is not something that folks want in their .toml and .rb files. T_T Just a comment. No change needed.

Reply:
Comma away